.github/workflows/nightly-doc-build.yml

```diff
@@ -5,6 +5,10 @@ on:
     - cron:  '0 4 * * *'
   workflow_dispatch:
 
+env:
+  DOCUMENTATION_CNAME: 'parametric.fluent.docs.pyansys.com'
```

Comment:
Pinging here @mkundu1 and @alikasim72.

This CNAME does not exist yet. We need to create it and use it in favor of "fluentparametric.docs.pyansys.com".

I would suggest to proceed as follows:

1. Manually copy all the content from [pyfluent-parametric-dev-docs](https://github.com/pyansys/pyfluent-parametric-dev-docs) in the `dev/` folder of the `gh-pages` branch of this repository.

2. Manually copy all the content from [pyfluent-parametric-docs](https://github.com/pyansys/pyfluent-parametric-docs) in the `release/X.Y` folder of the `gh-pages` branch of this repository.

3. Ping me so I can create the CNAME and link it to this repository.

4. Merge this pull-request, completing the migration.

This tasks apply for the rest of the repositories. It is a way of speeding up the migration process.

Reply:
@jorgepiloto the suggested changes have been made and the docs have been copied to their respective folders in the gh-pages branch.